code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -516,16 +490,16 @@
 			else
 				filtering = FALSE
 		if(filtering && moderator_internal.gases[filter_type])
-			var/datum/gas_mixture/removed = moderator_internal.remove_specific(filter_type, 20 * delta_time)
+			var/datum/gas_mixture/removed = moderator_internal.remove_specific(filter_type, 100 * delta_time)
 			if(removed)
 				linked_output.airs[1].merge(removed)
 
 		var/datum/gas_mixture/internal_remove
-		if(internal_fusion.gases[/datum/gas/helium][MOLES] > 0)
-			internal_remove = internal_fusion.remove_specific(/datum/gas/helium, internal_fusion.gases[/datum/gas/helium][MOLES] * (1 - (1 - 0.5) ** delta_time))
+		if(internal_fusion.gases[selected_fuel.requirements[3]][MOLES] > 0)
+			internal_remove = internal_fusion.remove_specific(selected_fuel.requirements[3], internal_fusion.gases[selected_fuel.requirements[3]][MOLES] * 0.95)
 			linked_output.airs[1].merge(internal_remove)
 		if(internal_fusion.gases[/datum/gas/antinoblium][MOLES] > 0)
-			internal_remove = internal_fusion.remove_specific(/datum/gas/antinoblium, internal_fusion.gases[/datum/gas/antinoblium][MOLES] * (1 - (1 - 0.05) ** delta_time))
+			internal_remove = internal_fusion.remove_specific(/datum/gas/antinoblium, internal_fusion.gases[/datum/gas/antinoblium][MOLES] * 0.95)
 			linked_output.airs[1].merge(internal_remove)
```

Comment:
Were these meant to no longer respect `delta_time`? Was changing these from 5% and 50% per second to 95% per update intentional?

Reply:
oh yeah i was doing some testing and i needed the moles to be removed much faster, i'll revert this one